### DISCUSSION TITLE
Is there a hyperkey complex modification?

### DISCUSSION THREAD

**Discussion by leblaze** [pqrs-org/KE-complex_modifications#1262] [Q&A] [answered]
There is this cool app called hyperkey, which lets you map right command to be command + option + control + shift so that you can have extra shortcuts. Is there a way to do this in KE?

**Comment by gsinti** [comment, marked as answer]
Take a look here: https://ke-complex-modifications.pqrs.org/?q=hyper%20key

These are made by members of the community and can be imported into KE.

You can also [make your own](https://github.com/pqrs-org/KE-complex_modifications#add-rules) to customize which key you'd prefer to use as the Hyper Key.
